libs/ui-toolkit/src/components/nav/trading/index.ts

```diff
@@ -0,0 +1 @@
+export * from './trading-nav';
```

Comment:
chore (blocking): As discussed lets move this to apps/trading. We can move it back when it actually becomes shared between apps.